extensions/vscode/package.json

```
@@ -135,6 +135,11 @@
         "title": "Select Active Configuration For Deployment",
         "category": "Posit Publisher"
       },
+      {
+        "command": "posit.publisher.homeView.showAssociateDeployment",
```

Nit to use verb language for the command, feel free to ignore.

```suggestion
        "command": "posit.publisher.homeView.associateDeployment",
```

---

sounds like a good suggestion. Thanks.

---

Updated.